modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -857,10 +978,35 @@ Description of the returned code::
 The hinted index does not exist, please check the schema (index is: INDEX FOR ()-[`r`:`Rel`]-() ON (`r`.`id`))
 
 Suggestions for improvement::
-The hinted index does not exist, make sure the label and property are spelled correctly.
+The hinted index does not exist, make sure the relationship type and property are spelled correctly.
 If the spelling is correct, either create the index or remove the hint from the query.
 
-====
+======
+[.include-with-GQLSTATUS-code]
+======
+
+Query::
++
+[source, cypher]
+----
+MATCH ()-[r:Rel]-()
+USING INDEX r:Rel(id)
+WHERE r.id = 1
+RETURN r
+----
+
+Returned GQLSTATUS code::
+01N31
+
+Returned Status Description::
+warn: hinted index not found.
+Unable to create a plan with `INDEX r:Rel(id)` because the index does not exist.
```

Comment:
```suggestion
Unable to create a plan with `INDEX :Rel(id)` because the index does not exist.
```

In the code it looks like the variable is not present here